packages/wrangler/src/publish.ts

```diff
@@ -78,13 +78,24 @@ export default async function publish(props: Props): Promise<void> {
     'You need to provide a name when publishing a worker. Either pass it as a cli arg with `--name <name>` or in your config file as `name = "<name>"`'
   );
 
+  if (config.site?.["entry-point"]) {
+    console.warn(
+      "Deprecation notice: The `site.entry-point` config field is no longer used.\n" +
+        "The entry-point should be specified via the command line (see `wrangler publish --help`) or the `build.upload.main` config field.\n" +
```

Comment:
same as above, maybe we can use a pseudo path here 
```suggestion
        "The entry-point should be specified via the command line as `wrangler publish path/to/entry` or the `build.upload.main` config field.\n" +
```